decidim-core/app/models/decidim/organization.rb

```diff
@@ -24,7 +24,7 @@ class Organization < ApplicationRecord
     validates :reference_prefix, presence: true
     validates :default_locale, inclusion: { in: :available_locales }
 
-    mount_uploader :homepage_image, Decidim::HomepageImageUploader
+    mount_uploader :homepage_image, Decidim::HomepageImageUploader # DEPRECATED
```

Comment:
Why isn't this removed?

Reply:
Because I need to access this uploader from the migration. Maybe I can remove this from here and reopen the class in the migration, add the `mount_uploader` part there and keep this clean?